lookout/style/format/feature_extractor.py

```diff
@@ -732,11 +733,9 @@ def _compute_labels_mappings(self, vnodes: Iterable[VirtualNode]) -> None:
             if vnode.y is not None:
                 support[vnode.y] += 1
 
-        for class_seq in sorted(support, key=lambda l: -support[l]):
-            # The idea of sorting is to have labels arranged from the most frequent to the less
-            if support[class_seq] < self.cutoff_label_support:
-                break
-        self.labels_to_class_sequences = [key for key, val in support.items()
+        self.labels_to_class_sequences = [key for key, val in sorted(support.items(),
+                                                                     key=itemgetter(1),
+                                                                     reverse=True)
```

Comment:
yeah, a mess happened here. thank you👍 
Just let keep the comment and make it not so right shifted?

```python
        # The idea of sorting is to have labels arranged from the most frequent to the less
        self.labels_to_class_sequences = [
            key for key, val in sorted(support.items(), key=itemgetter(1), reverse=True)
            if val >= self.cutoff_label_support]
```